### PR TITLE
Optional Bootstrap File Support

### LINKED ISSUE #61: Support for Initializing Pessimism Service with an Optional Configuration File
## Problem
Currently, Pessimism starts with a blank slate and requires clients to teach it what to monitor. This process might involve lengthy client interaction logic for deploying and initializing the service.

## Problem Solution
Support an optional initialization file that defines arbitrary invariant schemas. These schemas would be executed upon service start, similar to how a blockchain node ingests a genesis state during bootstrapping. This feature aims to provide a more seamless operationalization of the service via multiple users.

### PR DESCRIPTION
**Pull request by epociask**
## Fixes Issue

Fixes #61 
## Changes proposed
* Updated ID names to be less verbose and more appropriate
* Added boostraping configurability where the application can take a JSON file that's ingested to spawn invariant sessions at bootstrap 

## To Test
Create some `json` file at root directory level, populate like so:
```
[
    {
          "network": "layer1",
          "pipeline_type": "live",
          "type": "contract_event", 
          "start_height": null,
          "alert_destination": "slack",
          "invariant_params": {
              "address": "0xfC0157aA4F5DB7177830ACddB3D5a9BB5BE9cc5e",
              "args": ["Transfer(address, address, uint256)"]
        }
    },
    {
        "network": "layer1",
        "pipeline_type": "live",
        "type": "balance_enforcement", 
        "start_height": null,
        "alert_destination": "slack",
        "invariant_params": {
            "address": "0xfC0157aA4F5DB7177830ACddB3D5a9BB5BE9cc5e",
            "lower": 1,
            "upper": 2
       }
    }
]
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,40 @@ Unit tests can ran using the following project level command(s):
...
+
+### Example
+```
+[
+    {
+          "network": "layer1",
+          "pipeline_type": "live",
+          "type": "contract_event", 
+          "start_height": null,
+          "alert_destination": "slack",
```
Comment (a-drain) [inline README.md:67]: Should alert_destination be an array of strings? What if I want multiple alert destinations for the same invariant?

File: cmd/pessimism/main.go [hunk 2]
```diff
@@ -97,15 +97,32 @@ func main() {
 		os.Exit(1)
 	}
 
-	pess := app.New(ctx, m, srver)
+	pessimism := app.New(ctx, cfg, m, srver)
```
Comment (a-drain) [inline cmd/pessimism/main.go:100]: why not server

**Review by a-drain — COMMENTED** [review]
Just a few minor comments, otherwise LGTM

File: cmd/pessimism/main.go (hunk 2 above)
Comment (epociask) [reply cmd/pessimism/main.go:100]: Because the http server is a field of the application

File: README.md (hunk 1 above)
Comment (epociask) [reply README.md:67]: That's a great call-out! I feel like that change falls out-of-scope of this PR though. I can cut an issue in the meantime detailing the change.

Comment (epociask) [reply README.md:67]: That's a great call-out! I feel like that change falls out-of-scope of this PR though. I can cut an issue in the meantime detailing the change.

Comment (epociask) [reply README.md:67]: #81

**Review by a-drain — DISMISSED** [review]

**Review by a-drain — APPROVED** [review]

**Pull request merged**